app/src/app/components/contents/packets/PacketDetails.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect} from "react";
+import {useSelector} from "react-redux";
+import {RootState, useAppDispatch} from "../../../../types";
+import ParameterList from "./ParameterList";
+import {useParams} from "react-router-dom";
+import {actions} from "../../../store/packets/thunks";
+
+export default function PacketDetails() {
+
+    const dispatch = useAppDispatch();
+
+    const {packetId} = useParams();
+
+    useEffect(() => {
+        if (packetId) {
+            dispatch(actions.fetchPacketById(packetId));
+        }
+    }, [packetId]);
+
+    const {packet} = useSelector((state: RootState) => state.packets);
+
+    const paramLen = (): boolean => {
+        return Object.entries(packet.parameters).length > 0;
+    };
+
+    if (Object.keys(packet).length === 0) {
+        return <div>Loading...</div>;
+    }
```

Comment:
I guess at some point we'll have to deal with the cases which will never complete 'Loading':
1. `packetId` is not set in the route so we never attempt to fetch packet
2. `packetId` is not valid (dealing with the 404 response to be added in mrc-4331)

Could deal with 1. now or in another ticket.